content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst

```diff
@@ -117,14 +113,46 @@ serves as an additional tool to update the unit price of products.
    :alt: Add manual valuation of stock value to a product.
 
 Create manual valuation entries on the :guilabel:`Stock Valuation` dashboard by first navigating to
-:menuselection:`Inventory app --> Reporting --> Inventory Valuation`, and then click the little gray
-drop-down icon to the left of a product's name, to then reveal stock valuation line items below as
-well as a teal :guilabel:`+ (plus)` button on the right.
+:menuselection:`Inventory app --> Reporting --> Valuation`. Next, to enable the product revaluation
+button, select :menuselection:`Group by --> Product` to organize all the records by product. Click
+on the gray drop-down icon to reveal stock valuation line items below, as well as a teal
```

Comment:
I believe this "dropdown icon" would require a :guilabel: since we are instructing the user to click on it and describing it visually
```suggestion
on the gray :guilabel:`drop-down` icon to reveal stock valuation line items below, as well as a teal
```

Reply:
You're right about the guilabel! What do you think about taking it further and turning it into an emoji?
```suggestion
on the gray :guilabel:`🔽 (drop-down)` icon to reveal stock valuation line items below, as well as a teal
```